subprojects/distributions-dependencies/build.gradle.kts

```diff
@@ -57,6 +57,7 @@ dependencies {
         api(libs.commonsLang)           { version { strictly("2.6") }}
         api(libs.commonsLang3)          { version { strictly("3.12.0") }}
         api(libs.commonsMath)           { version { strictly("3.6.1") }}
+        api(libs.configurationCacheReport)          { version { strictly("1.0") }}
```

Comment:
I wonder if the configuration cache report versions should somehow be aligned with the Gradle version.

Reply:
Given it changes very rarely, aligning versions would add some overhead. By the way, what's the process to publish a new version?

Reply:
In any case, `configuration-cache-report` is definitely *NOT* an `api` dependency.

Reply:
It's removed from `api`. The check and release process is automated here: https://builds.gradle.org/project.html?projectId=Gradle_ConfigurationCacheReport&branch_Gradle_ConfigurationCacheReport=%3Cdefault%3E